README.NewResourceFromTerraform.md

```diff
@@ -615,6 +615,11 @@ update to date, we need to add the new CRDs and samples to existing doc.
     `https://cloud.google.com` (i.e.
     `https://cloud.google.com/spanner/docs/reference/rest/v1/projects.instances`).
     Add an additional paragraph/table if needed.
+1.  Update the URL in "Before you begin" section, points to the "Prerequisites" or 
```

Comment:
This "Before you begin" section can't be found in a random existing file (as suggested in bullet point #1), so would be great that an example PR is linked here.

Reply:
Updated